apps/web/pages/sitemap.xml.ts

```diff
@@ -0,0 +1,383 @@
+import { Client, gql, fetchExchange, cacheExchange } from "@urql/core";
+import { globby } from "globby";
+import type { GetServerSidePropsContext } from "next";
+import { withAxiomGetServerSideProps } from "next-axiom";
+// import { getStaticPaths as getStaticPathsResources } from "pages/scheduling/[...slugs]";
+import { z } from "zod";
+
+import { prisma } from "@calcom/prisma";
+
+// import { getAllBlogPaths } from "@lib/blog/server";
+
+const locales = [
+  "en",
+  "es",
+  "de",
+  "nl",
+  "pl",
+  "pt-BR",
+  "sr",
+  "tr",
+  "vi",
+  "zh-CN",
+  "zh-TW",
+  "fr",
+  "it",
+  "ar",
+  "cs",
+  "pt",
+];
+export enum SiteLocale {
+  Ar = "ar",
+  De = "de",
+  En = "en",
+  Es = "es",
+  Fr = "fr",
+  It = "it",
+  Pt = "pt",
+}
+const urql = new Client({
+  url: "https://graphql.datocms.com/",
+  exchanges: [cacheExchange, fetchExchange],
+  fetchOptions: () => {
+    if (!process.env.DATOCMS_API_TOKEN) {
```

Comment:
I'm not too sure if we should bring DATOCMS into webapp. AFAIK this is a website only thing. cc @keithwillcode @emrysal

Reply:
Yeah I don’t think we should